tools/index.js

```diff
@@ -0,0 +1,745 @@
+!function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
We shouldn't commit that file

Reply:
Also, I prefer that we copy the way features are implemented:
 - added to specific folder in dist with modularized ES5 (with import/export)
 - simple file in tools index.js:
   ```js
   export { MediaCapabilitiesProber } from "../dist/_esm5.minimal/tools";
   ```

I think that the first one is already done when running:
```sh
./scripts/generate_minimal_build
```

Reply:
Moreover, isn't it an experimental feature?